features/collections/collection_anonymity.feature

```diff
@@ -4,422 +4,244 @@ Feature: Collection
   As a humble user
   I want to create a collection with anonymity and hidden-until-reveal works
 
```

Comment:
Note to reviewers: I suggest just reading the current version of this feature as it has been completely rewritten and so the diff is not very helpful!